addons/hr/models/hr_employee.py

```diff
@@ -597,11 +597,11 @@ def _get_calendar_attendances(self, date_from, date_to):
 
     def _get_marital_status_selection(self):
         return [
-            ('single', 'Single'),
-            ('married', 'Married'),
-            ('cohabitant', 'Legal Cohabitant'),
-            ('widower', 'Widower'),
-            ('divorced', 'Divorced')
+            (_('single'), _('Single')),
+            (_('married'), _('Married')),
+            (_('cohabitant'), _('Legal Cohabitant')),
+            (_('widower'), _('Widower')),
+            (_('divorced'), _('Divorced'))
```

Comment:
```suggestion
            ('single', _('Single')),
            ('married', _('Married')),
            ('cohabitant', _('Legal Cohabitant')),
            ('widower', _('Widower')),
            ('divorced', _('Divorced'))
```
We don't want to translate the first item in the tuple, since that is the key that is saved in the field. 😉